source/auth/auth.rst

```diff
@@ -352,7 +401,14 @@ SCRAM-SHA-1
 
 SCRAM-SHA-1 is defined in `RFC 5802 <http://tools.ietf.org/html/rfc5802>`_.
 
-`Page 8 of the RFC <http://tools.ietf.org/html/rfc5802#page-8>`_ identifies the "SaltedPassword" as ``:= Hi(Normalize(password), salt, i)``. The ``password`` variable MUST be the mongodb hashed variant. The mongo hashed variant is computed as :javascript:`hash = HEX( MD5( UTF8( username + ':mongo:' + plain_text_password )))`, where ``plain_text_password`` is actually plain text. For example, to compute the ClientKey according to the RFC:
+`Page 8 of the RFC <http://tools.ietf.org/html/rfc5802#page-8>`_ identifies the
+"SaltedPassword" as ``:= Hi(Normalize(password), salt, i)``. The ``password``
+variable MUST be the mongodb hashed variant. The mongo hashed variant is
+computed as :javascript:`hash = HEX( MD5( UTF8( username + ':mongo:' +
+plain_text_password )))`, where ``plain_text_password`` is actually plain text.
+The ``password`` MUST NOT be prepared with SASLprep before hashing.
```

Comment:
The username and password MUST NOT...

Reply:
Done.  Also moved the username paragraph below up higher so this flows more naturally.